cloudflare/tests/cf_tests.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {describe, test, expect} from "vitest"
+import {readFile} from "fs/promises"
+import {mf} from "./mf"
+
+describe("fetch", () => {
+  test("loadfiles", async () => {
+    let placeholder = (await readFile("../examples/jsonplaceholder.graphql")).toString()
+    let placeholder_batch = (await readFile("../examples/jsonplaceholder_batch.graphql")).toString()
+
+    let bucket = await mf.getR2Bucket("MY_R2")
+    // @ts-ignore
+    await bucket.put("examples/jsonplaceholder.graphql", placeholder)
+    // @ts-ignore
+    await bucket.put("examples/jsonplaceholder_batch.graphql", placeholder_batch)
+  })
+  test("ide", async () => {
+    let resp = await mf.dispatchFetch("https://fake.host/", {
+      method: "GET",
+    })
+    let body = await resp.text()
+    expect(body.includes("<title>Tailcall - GraphQL IDE</title>")).toBe(true)
+    expect(resp.status).toBe(200)
+  })
+
+  test("sample_resp", async () => {
+    let resp = await mf.dispatchFetch("https://fake.host/graphql?config=examples/jsonplaceholder.graphql", {
+      method: "POST",
+      body: '{"query":"{\\n  user(id: 1) {\\n    id\\n  }\\n}\\n"}',
```

Comment:
the `\n` are not necessary here. Could be easier to read if we remove it. Also we can use different way to specify it like template strings or JSON.stringify for big requests